e2e/app/modal/share-modal.ts

```diff
@@ -57,12 +57,18 @@ export default class ShareModal extends Modal {
 
     const typeAndAddUser = async (name: string): Promise<boolean> => {
       const nameWithoutDomain = name.split('@')[0];
-      for (const char of nameWithoutDomain) {
+      // Split string into segments of 4 characters. Type 4 chars at a time, much faster than type 1 char and check.
+      const n = 4;
+      for (let i = 0, charsLength = nameWithoutDomain.length; i < charsLength; i += n) {
+        const chars = nameWithoutDomain.substring(i, i + n);
         const input = await this.waitForSearchBox().asElementHandle();
-        const waitForResponsePromise = this.page.waitForResponse((response) => {
-          return response.url().includes('/userSearch/registered/') && response.request().method() === 'GET';
-        });
-        await input.type(char);
+        const waitForResponsePromise = this.page.waitForResponse(
+          (response) => {
+            return response.url().includes('/userSearch/registered/') && response.request().method() === 'GET';
+          },
+          { timeout: 60000 }
```

Comment:
Unrelated but just saw a new test failure [here](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/59652a71c9e77c0001f87081/611e8ec7f724025bffab2006-0-build/artifacts/logs/jest/workspace-share-modal.spec.FAILED.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210819T181948Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210819%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=da4a4311eb395238f0b809c1b8289dd614d119a6f0695602a04461e6713e6515). 
Adding new `timeout` parameter in `waitForResponse` function because function has failed due to a very long request time.

`yarn lint --fix` updated formatting afterward.